pytorch_lightning/plugins/__init__.py

```diff
@@ -9,10 +9,10 @@
     TrainingTypePluginsRegistry,
 )
 from pytorch_lightning.plugins.precision.apex_amp import ApexMixedPrecisionPlugin
-from pytorch_lightning.plugins.precision.deepspeed_precision import DeepSpeedPrecisionPlugin
```

Comment:
You also need to rename the `deepspeed_precision.py` and `ipu_precision.py` files